cmd/alias-remove.go

```diff
@@ -80,11 +81,24 @@ func mainAliasRemove(ctx *cli.Context, deprecated bool) error {
 	return nil
 }
 
+// isAliasIn is a helper function that checks if a given alias is present in Aliases array, returns error if not found
+func isAliasIn(alias string) *probe.Error {
+	hostConfig := mustGetHostConfig(alias)
+	if hostConfig == nil {
+		fatalIf(errInvalidAliasedURL(alias), "No such alias `"+alias+"` found.")
+	}
+	return nil
+}
+
 // removeAlias - removes an alias.
 func removeAlias(alias string) aliasMessage {
 	conf, err := loadMcConfig()
 	fatalIf(err.Trace(globalMCConfigVersion), "Unable to load config version `"+globalMCConfigVersion+"`.")
 
+	// check if alias is valid
+	err = isAliasIn(alias)
+
+	fatalIf(err.Trace(alias), alias+" not found in the configuration file. Please confirm the alias name you are trying to remove. You can use mc alias list to view a list of all aliases configured.")
```

Comment:
we already exit inside isAliasIn() there is no reason for this function to return a `nil` and this fatalIf() will never be called.

```
func aliasMustExist(alias string) {
hostConfig := mustGetHostConfig(alias)
	if hostConfig == nil {
		fatalIf(errInvalidAliasedURL(alias), "No such alias `"+alias+"` found.")
}
```

This is all we need

Reply:
@jinapurapu